converter/internal/promtailconvert/promtailconvert.go

```diff
@@ -147,9 +168,9 @@ func appendScrapeConfig(
 	// Append all the components that produce logs directly.
 	// If any relabelling is required, it will be done via a loki.relabel component.
 	// The logs are sent to loki.process if processing is needed, or directly to loki.write components.
-	//TODO(thampiotr): add support for other integrations
 	b.AppendCloudFlareConfig()
 	b.AppendJournalConfig()
+	b.AppendPushAPI()
 }
 
 func newLokiWrite(client *client.Config, diags *diag.Diagnostics, index int) (*builder.Block, loki.LogsReceiver) {
```

Comment:
Is it time to move newLokiWrite and toLokiWriteArguments to a component file in internal/build?

Reply:
Yes, 100% - somehow I've missed that 😅